pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/LocalPinotFS.java

```diff
@@ -40,6 +40,11 @@ public LocalPinotFS() {
   public void init(Configuration configuration) {
   }
 
+  @Override
+  public void mkdir(URI uri) throws IOException {
+    FileUtils.forceMkdir(new File(uri));
```

Comment:
What's the behavior for forceMkdir if the directory already exists? We should not delete existing directory.

Reply:
Agree that we should collect all necessary apis, but I missed this one. I think after this one, we should be complete. forceMkDir is what we currently use in our code, so I wanted to keep the behavior the same. It doesn't overwrite the directory if it already exists.